test/stored_queries/test_ncbi_tax.py

```diff
@@ -136,3 +136,52 @@ def test_siblings_nonexistent_node(self):
             data=json.dumps({'key': 'xyz'}),  # Nonexistent node
         ).json()
         self.assertEqual(resp['count'], 0)
+
+    def test_search_sciname_prefix(self):
+        """Test a query to search sciname."""
+        resp = requests.post(
+            _CONF['re_api_url'] + '/api/v1/query_results',
+            params={'stored_query': 'ncbi_taxon_search_sci_name'},
+            data=json.dumps({'search_text': 'prefix:bact'}),
```

Comment:
What other modifiers like this are possible? Are there any that could cause problems, given we're basically exposing this part of the arango api?

Reply:
The prefixes are "prefix:xyz", "complete:xyz", (the default), "|xyz" (or operator), "-xyz" (negation), "+xyz" (the default). I really don't think there would be any problems exposing this